test/test_sparse.py

```diff
@@ -3234,7 +3219,7 @@ def sparse_log(x):
                   *get_all_fp_dtypes(
                       include_half=(CUDA11OrLater and SM53OrLater),
                       include_bfloat16=(CUDA11OrLater and SM80OrLater)))
-    @precisionOverride({torch.bfloat16: 1e-2, torch.float16: 1e-2, torch.complex64: 1e-2, torch.float32: 1e-2})
+    @precisionOverride({torch.bfloat16: 5e-2, torch.float16: 5e-2, torch.complex64: 1e-2, torch.float32: 1e-2})
```

Comment:
Why is the decreased precision necessary? It seems an unrelated change.

Reply:
After enabling more tests in this PR, the seed to the random tensor generator changed leading to different inputs to this test function. The new inputs required a decrease of precision here (IIRC, the difference reported was about 2.2e-2).

Reply:
Ok, will do it.

Reply:
Restored the original state as the tests pass again with the given precision parametes.